modules/alloy/src/main/ts/ephox/alloy/behaviour/positioning/PositionApis.ts

```diff
@@ -63,7 +63,9 @@ const positionWithinBounds = (component: AlloyComponent, posConfig: PositioningC
     // We need to calculate the origin (esp. the bounding client rect) *after* we have done
     // all the preprocessing of the component and placee. Otherwise, the relative positions
     // (bottom and right) will be using the wrong dimensions
-    const origin = posConfig.useFixed() ? getFixedOrigin() : getRelativeOrigin(component);
+    const origin = posConfig.usePositioningType() === 'fixed'
```

Comment:
Instead of doing an `=== 'fixed'` check everywhere, we should be an `isFixed` function somewhere. It might make sense to put it in `Positioning` as one of its ExtraApis that just takes in config

Reply:
We would be able to call it in the `getMode` function below as well.

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8294/commits/3f69566aaac7634750c0a703b7eff61a9e091b16)